recipes/rhash/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+make ADDCFLAGS="${CFLAGS}" ADDLDFLAGS="${LDFLAGS}" build-shared
+
+make PREFIX="" DESTDIR=${PREFIX} install-shared
+
+make -C librhash PREFIX="" DESTDIR=${PREFIX} install-headers install-lib-shared install-lib-static
+
+cd tests
+./test_rhash.sh
```

Comment:
What if we change this line to `bash -e test_rhash.sh`? That should make sure the test script exits on any failure. If that works, it might be worth adding a one line PR to upstream that adds `set -e` to this script.